workflow/config_for_tests.yml

```diff
@@ -31,7 +31,7 @@ ValidationConfig:
             dataset_description: ../workflow/dataset_for_tests_java.json
         SVMSInfo:
           - name: usvm
-            launch_command: java -jar app.jar --mode server --port {port}
+            launch_command: java -jar -DLOG_DIR=$PWD app.jar --mode server --port {port}
```

Comment:
Why can't we just run it with `-DLOG_DIR=./`?

Reply:
`./` is a usvm subfolder containing `app.jar`, so it will look like `./../../../../../AIAgent/`, which is terrible and it will force you to write like this for each svm